jinx.el

```diff
@@ -825,6 +863,30 @@ With prefix argument GLOBAL change the languages globally."
   (jinx--load-dicts)
   (jinx--cleanup))
 
+;;;###autoload
+(defun jinx-correct-visible ()
+  "Correct visibly misspelled words in current window."
+  (interactive)
+  (unless jinx-mode (jinx-mode 1))
+  (let* ((old-point (point-marker)))
+    (unwind-protect
+        (jinx--correct-overlays
+         (jinx--force-overlays (window-start) (window-end) :visible t))
+      (goto-char old-point)
+      (jinx--in-base-buffer #'jit-lock-refontify (window-start) (window-end)))))
```

Comment:
Could we could move more code into `jinx--correct-overlays` to reduce the duplication with `jinx-correct-buffer`?